docs/source/workflows/predictors.rst

```diff
@@ -133,6 +134,79 @@ The following example demonstrates how to create an :class:`~citrine.informatics
+The features are configured using the ``features`` and ``excludes`` arguments, which accept either feature names or predefined aliases.
+The default is the `standard` alias, corresponding to eight features that are a good balance of cost and performance.
+
+The feature names and descriptors are automatically constructed from the name of the input and the feature names.
+The ``from_predictor_responses`` method will grab the descriptors for the features so that they can be fed into other predicors,
+e.g. the :class:`~citrine.informatics.predictors.auto_ml_predictor.AutoMLPredictor`, as inputs.
+
+
+The following example demonstrates how to use a :class:`~citrine.informatics.predictors.molecular_structure_featurizer.MolecularStructureFeaturizer` and
+:class:`~citrine.informatics.predictors.auto_ml_predictor.AutoMLPredictor` to model a property of a molecule:
+
+.. code:: python
+
+    from citrine.informatics.descriptors import MolecularStructureDescriptor, RealDescriptor
+    from citrine.informatics.predictors import MolecularStructureFeaturizer, AutoMLPredictor, GraphPredictor
+    from citrine.seeding.find_or_create import create_or_update
+    from citrine.informatics.data_sources import GemTableDataSource
+
+
+    # descriptor for the molecular structure input
+    input_desc = MolecularStructureDescriptor('Solvent SMILES')
+    # descriptor for the property to define a machine learning model to predict
+    output_desc = RealDescriptor(
+        key="density",
+        units="g/cm^3",
+        lower_bound=0.0,
+        upper_bound=100.0
+    )
+
+
+    # featurize the molecular structure
+    featurizer = MolecularStructureFeaturizer(
+        name='Molecular Featurizer',
+        description="Featurizer the Solvent's molecular structure using the default features.",
+        descriptor=input_desc,
+        features=['standard'],
+    )
+
+    # get the feature names
+    features = project.descriptors.from_predictor_responses(
+        predictor=featurizer,
+        inputs=[input_desc]
+    )
+ 
+    # create AutoMLPredictor, using the feature names as inputs
+    # note: the molecular structure, `input_desc`, should not be included in the inputs here
+    ml_predictor = AutoMLPredictor(
+        name='ML Model for Density',
+        description='Predict the density, given molecular features of the solvent',
+        inputs = features,
+        output = output_desc,
+        training_data = [GemTableDataSource(training_data_table_uid, 1)]
+    )
+ 
+    # use a graph predictor to wrap together the featurizer and the machine learning model
+    graph_predictor = GraphPredictor(
+        name='Density from solvent molecular structure',
+        description='Predict the density from the solvent molecular structure using molecular structure features.',
+        predictors = [featurizer, ml_predictor],
+        training_data = [GemTableDataSource(training_data_table_uid, 1)] # training data shared by all sub-predictors
```

Comment:
Why include the training data both here and in the AutoMLPredictor?

Reply:
:shrug: I'll take it out.